api-docs/components/schemas/null.json

```diff
@@ -1,6 +1,5 @@
 {
   "NullValue": {
-    "nullable": true,
```

Comment:
"nullable" is a property that must be attached to a type. We don't need it here b/c we're using `not anyOf` and where null is not excluded. See https://github.com/ajv-validator/ajv/issues/1948 for another example and explanation